cfme/scripting/link_config.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import click
+from pathlib2 import Path
+
+
+@click.command()
+@click.argument('src', type=click.Path(exists=True, file_okay=False))
+@click.argument('dest', type=click.Path(exists=True, file_okay=False))
+def main(src, dest):
+    """
+    links configfiles from one folder to another
+
+    if links exists it verifies content
+    if files exist at the target side it errors
+
+    Args:
+      @src: source folder
+      @dest: target folder
+    """
```

Comment:
**REQUIRED**: Comment styling is wrong, please fix -
 https://gist.github.com/mfalesni/11694e0d563c80c3917ce6012400b93f#file-example-py-L22 or http://cfme-tests.readthedocs.io/guides/dev_guide.html